src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -110,3 +110,12 @@ def _make_tags(replay_id, url, replay_event):
         tags.update(replay_event["tags"])
 
     return tags
+
+
+def _make_clicked_element(node):
+    element = node.get("tagName", "")
+    if "attributes" in node:
+        for key, value in node["attributes"].items():
+            element += "[" + key + '="' + value + '"]'
```

Comment:
We should represent ids with a `#` and classes with a `.`.  E.g. `a#id.class.class2[attribute="a"]`